test/runtests.jl

```diff
@@ -335,11 +337,11 @@ end
     ft = ftest(mod, nullmod)
     @test isapprox(ft.cols[end][2],  2.481215056713184e-8)
     # Test output
-    @test sprint(show, ftest(mod, nullmod)) == "         Residual DOF ΔResidual DOF DOF Δdof      SSR    ΔSSR           R²       ΔR²      F*      p(>F)\nModel 1          10.0           NaN 3.0  NaN 0.128333     NaN     0.960258       NaN     NaN        NaN\nModel 2          11.0           1.0 2.0 -1.0  3.22917 3.10083 -2.22045e-16 -0.960258 241.623 2.48122e-8\n"
+    @test sprint(show, ftest(mod, nullmod)) == """         Res. DOF DOF ΔDOF      SSR     ΔSSR           R²      ΔR²      F*      p(>F)\nModel 1      10.0 3.0  NaN 0.128333      NaN     0.960258      NaN     NaN        NaN\nModel 2      11.0 2.0  1.0  3.22917 -3.10083 -2.22045e-16 0.960258 241.623 2.48122e-8\n"""
```

Comment:
Actually, what I meant is that with triple-quoted string you can replace `\n` with actual line breaks (see the manual). Also add a line break before `"""`.